lizmap/modules/action/classes/actionDatasource.class.php

```diff
@@ -41,11 +41,11 @@ public function __construct($repository, $project, $layerId)
         jClasses::inc('action~actionConfig');
         $dv = new actionConfig($repository, $project);
         if (!$dv->getStatus()) {
-            return $this->error($dv->getErrors());
+            return $dv->getErrors();
```

Comment:
@rldhont **Phpstan** say  the `error` method doesn't exists, so i think we need to return `$dv->geErrors()` directly or define `$this->errors = $dv->geErrors()` and return this ?

Reply:
@mdouchin can you confirm ?

Reply:
I confirm the proposed fix is OK. By the way, it seems the whole class actionDatasource is never used in Lizmap ! I will review this module code for next point release !